package.json

```diff
@@ -1,5 +1,5 @@
 {
-	"name": "rocket.chat.livechat",
+	"name": "@rocket.chat/livechat",
```

Comment:
It would be better naming this package as `enterprise` because we have plans to have a community version in a near feature.
`@rocket.chat/livechat-enterprise` or `@rocket.chat/livechat.enterprise`... something like this.